netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -180,6 +181,18 @@ public NettyServerBuilder sslContext(SslContext sslContext) {
   }
 
   /**
+   * Sets the {@link ProtocolNegotiator} to be used. If non-{@code null}, overrides the value
+   * specified in {@link #sslContext(SslContext)}.
+   *
+   * <p>Default: {@code null}.
+   */
+  public final NettyServerBuilder protocolNegotiator(
```

Comment:
I don't think it's less intended for use than any of the other fields on the builders. It's essentially part of the builder API now and should assume the same strictness of usage.